internal/module/conditions.go

```diff
@@ -10,9 +10,9 @@ import (
 )
```

Comment:
actually i don't remember exactly what we have agreed upon. I suggest we leave it here for now, and it will be removed once the condition/status implementation of the CR will be delivered